dev/deps/spark-deps-hadoop-2.6

```diff
@@ -182,7 +182,7 @@ slf4j-api-1.7.16.jar
 slf4j-log4j12-1.7.16.jar
 snakeyaml-1.15.jar
 snappy-0.2.jar
-snappy-java-1.1.2.6.jar
+snappy-java-1.1.4.jar
```

Comment:
Is there any reason to use `1.1.4` instead of `1.1.7.1`?

Reply:
+1